### ISSUE TITLE
Emulator does not confirm phone verification

### ISSUE THREAD

**Issue by pseudobacon** [MarcoG3/WhatsDump#2]
Hi this is a very neat tool but I'm running into an issue where the OK button is not being pressed to confirm the mobile number to be verified. This results in a message saying the phone was registered successfully before the verification step. 

![Verifying](https://user-images.githubusercontent.com/12183854/57205152-defd5a00-6fff-11e9-9091-fe7f9c266612.png)

**Comment by MarcoG3** [comment]
Should be fixed in c1f2746, if you still have that issue or any other one please run the script with `--verbose` and paste the full output so it is easier to debug, thanks!

**Comment by pseudobacon** [comment]
It worked only once, tried a couple more times and it still seems to give up too early

![image](https://user-images.githubusercontent.com/12183854/57217362-c60f9b80-7035-11e9-87e6-21e48d065dfa.png)

Also, when not on verbose theres an additional warning:
![image](https://user-images.githubusercontent.com/12183854/57217403-e3dd0080-7035-11e9-8c01-2ecba7c556a3.png)

**Comment by MarcoG3** [comment]
Yes, it looks like it is stuck in that dialog where a spinner is displayed. The most reliable way would be to reverse engineer WhatsApp and take a look at the timeout they put and replicate it in WhatsDump. I will take a look at this soon.

Meanwhile, could you give me an average estimate time it takes for that "spinner" dialog to close?

**Comment by MarcoG3** [comment]
As for the additional emulator warning, I don't really know what is the underlying issue. It could be some hardware acceleration specific warning or something related to android emulator.

**Comment by MarcoG3** [comment]
I've analyzed WhatsApp code and made some tests and it looks like that there is no timeout. Spinner bar dialog never closes if there is some connectivity issue. I've once again increased timeout from 2 to 5 minutes in ba290f4.

**Comment by pseudobacon** [comment]
Okay got a new one for you today 

![image](https://user-images.githubusercontent.com/12183854/57263407-7623e980-70b4-11e9-9a12-621e822bd562.png)

Looks like the verifying call needs a longer time out as well

I manually hit the restore button, but then ran out of space on the emulator. How do I go about increasing the internal storage size on the emulator?

![image](https://user-images.githubusercontent.com/12183854/57263720-ab303c00-70b4-11e9-9e63-8f1c2258009c.png)

**Comment by MarcoG3** [comment]
Good point. `emulator` bin accepts an argument `-partition-size` where you can specify size in MB. I'll make a commit soon to fix that as well.

**Comment by MarcoG3** [comment]
Adjusted restore button timeout in b893f3d and increased user data partition from 800 MB (default) to 2048 MB in cc68db0.

**Comment by pseudobacon** [comment]
Ok thats working now. But I have a new issue where its not recognising that the restore has completed
